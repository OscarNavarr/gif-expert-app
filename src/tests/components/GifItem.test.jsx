import {render} from '@testing-library/react';
import {GifItem} from '../../components/GifItem';

describe('Pruebas en <GifItem/>', () => { 

    const title = 'Warren Buffet';
    const url = 'https://codigo-buffetologia.com/codigoGraham.jpg';

    test('Debe de hacer match con el snapshop', () => { 

        const { container } = render( <GifItem title={ title } url={ url } /> );
        expect( container ).toMatchSnapshot();

     });
});