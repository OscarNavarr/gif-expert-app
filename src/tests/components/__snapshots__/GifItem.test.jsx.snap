// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifItem/> Debe de hacer match con el snapshop 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="Warren Buffet"
      src="https://codigo-buffetologia.com/codigoGraham.jpg"
    />
    <p>
      Warren Buffet
    </p>
  </div>
</div>
`;
